src/components/Wall/JoinedPickups.vue

```diff
@@ -33,7 +33,7 @@ export default {
   },
   data () {
     return {
-      showPickups: true,
+      showPickups: false,
```

Comment:
Me and @tiltec think showing the joined pickups by default is nice, maybe makes it more likely that people will do them :)

Reply:
Great :)